src/views/UserProfile.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div class="user-profile">
+    <h1>This is the profile view</h1>
+    <div class="content">
+       <ShowNotes></ShowNotes>
```

Comment:
uhmmmmm why u didnt iterate here over notes? this component is useless as u can see